Core/Inc/main.h

```diff
@@ -32,6 +32,9 @@ extern "C" {
 
 /* Private includes ----------------------------------------------------------*/
 /* USER CODE BEGIN Includes */
+#define SD_CS_GPIO_Port GPIOF  //SD card chip select GPIO Port
```

Comment:
Should these #defines be here or in SDCard.h?

Reply:
They should be here because the user_diskio_spi.c file needs access to these macros.

Reply:
I'm moving this to SDCard.h and adding `#include "SDCard.h"` to user_diskio_spi.c